src/modbus-tcp-client.js

```diff
@@ -10,6 +10,7 @@ module.exports = stampit()
             currentRequestId    = reqId,
             closedOnPurpose     = false,
             reconnect           = false,
+            buffer              = new Buffer(0),
```

Comment:
From [here](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_concat_list_totallength) I would say this is not specified so I would allocate an empty Buffer.